configuration/src/keyVaultConfiguration.ts

```diff
@@ -0,0 +1,72 @@
+/* Copyright (c) Microsoft Corporation. All Rights Reserved. */
+import * as fs from 'fs';
+import * as KeyVault from 'azure-keyvault';
+const AuthenticationContext = require('adal-node').AuthenticationContext;
```

Comment:
adal-node doesn't have typings, and we aren't using enough of adal-node to warrant creating typings for ourselves.